src/Templates/src/templates/maui-mobile/Resources/Styles/Styles.xaml

```diff
@@ -326,15 +326,15 @@
 
     <Style TargetType="Switch">
```

Comment:
Setting switch styles is very _hard_ because the switch control presents differently on different platforms, and because setting switch styles is also limited by (1) the inability to set track color when the switch is Off, and (2) the inability to set AppThemeBindings with OnPlatform bindings.

That being said, it's important that the enabled Switch controls have accessible color contrast ratios. (This does not apply to disabled controls however; disabled controls are typically grayed out (Android) or lower in opacity (iOS)).

Anyways, given the limitations, here's my suggestion for new styles we can apply here:

```suggestion
    <Style TargetType="Switch">
      <Setter Property="VisualStateManager.VisualStateGroups">
          <VisualStateGroupList>
              <VisualStateGroup x:Name="CommonStates">
                  <VisualState x:Name="Normal" />
                  <VisualState x:Name="Disabled">
                      <VisualState.Setters>
                          <Setter Property="OnColor" Value="{AppThemeBinding Light={StaticResource Gray300}, Dark={StaticResource Gray500}}" />
                          <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Gray200}, Dark={StaticResource Gray600}}" />
                      </VisualState.Setters>
                  </VisualState>
                  <VisualState x:Name="On">
                      <VisualState.Setters>
                          <Setter Property="OnColor" Value="{AppThemeBinding Light={StaticResource Secondary}, Dark={StaticResource Primary}}" />
                          <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Primary}, Dark={StaticResource Secondary}}" />
                      </VisualState.Setters>
                  </VisualState>
                  <VisualState x:Name="Off">
                      <VisualState.Setters>
                          <Setter Property="ThumbColor" Value="{AppThemeBinding Light={StaticResource Gray600}, Dark={StaticResource Gray200}}" />
                      </VisualState.Setters>
                  </VisualState>
              </VisualStateGroup>
          </VisualStateGroupList>
      </Setter>
    </Style>
```